database/populateReviews.js

```diff
@@ -0,0 +1,238 @@
+module.exports.reviewOptions = [
```

Comment:
I wouldn't change this now due to the cascading refactor it could cause, but you can set the module.exports object to directly be the array for a dummy data file like this.